pkg/plugin/plugin.go

```diff
@@ -125,7 +125,7 @@ func newUnmanagedPlugin(providerName string, config *plugin.ReattachConfig) (*un
 		return nil, fmt.Errorf("failed to cast plugin")
 	}
 	return &unmanagedPlugin{
-		name:     providerName,
+		name:     fmt.Sprintf("%s_%s", providerName, providerName),
```

Comment:
Do we leave this now in? Just want to make sure that this is not an oversight

Reply:
not an oversight